README.md

```diff
@@ -339,17 +362,11 @@ if (matchHashes(foundCertFingerprints, baseCertFingerprints)
 
 **Problem with this approach:**
 
-Since ``callingActivity?.packageName`` is only available from the
-activity that was originally called by ``startActivityForResult()``,
-the IDP app cannot change the activity before redirecting back to
-the RP app. That is a problem in cases where the IDP app uses different
-activities to log the user in or to request a 2-factor authentication.
-For this reason we do not recommend using ``startActivityForResult()``
-because it takes away flexibility and does only provide a negligible
-amount of additional security. The only thing we lose is the ability
-to determine which app has called the IDP app. But since we can trust the
-redirect uri, we can make a secure redirection with this uri and the
-mechanics described above.
+When an activity is launched with `startActivityForResult()` care is required when the iDP app launches further activities, such as sub-activities to request login or 2-factor authentication.
+
+The 1.2.0 release of androidx.activity provides APIs that simplify this process, though currently (August 2020) this version is still an alpha release.
```

Comment:
Can you explain what exactly improves the process with these APIs?

Reply:
It's the usual thing with the androidx library - it smooths out oddities in the process by providing a consistent API that works the same across all the version of Android out in the wild and tries to make things simpler.

I'm perhaps tempted to just reduce all the mention of `startActivityForResult()`, I'm dubious if there's ever a need to use it, and it's extra work for the iDP to implement it as they definitely already need to support the non-startActivityForResult() way for the web2app flow to work. Any thoughts?

Reply:
Should we mention AndroidX here? I understand that it makes programming easier, but if this is not related to our problem, isn't it a little bit of topic here? I mean if it would help to launch further activities in the IDP app I would say it is good but I did not find something about this on the website.

Concerning ``startActivityForResult()`` I would mention it in this README but not integrate it into the [API](https://github.com/oauthstuff/app2app-evolution/blob/master/AppAuth-Integration.md) of the AppAuth-Android project (I am currently preparing a pull request with the mechanics we discuss here). I think it does not improve security but makes the integration more complex. @danielfett we already talked about this but have you any further thoughts about this?

Reply:
+1 to what Fabian said. Mentioning `startActivityForResult()` is useful as a reference, but we should not use it in the AppAuth project. I don't think that we need to mention androidx.

Reply:
Yep, that's fine, I'll remove the androidx bit, and add the sentence I mentioned above explaining startActivityForResult() can't be used for web2app anyway.